docs/en/ecosystem_release/starrocks-kubernetes-operator.md

```diff
@@ -40,9 +40,62 @@ For example, the download URL for kube-starrocks chart v1.8.6 is:
 
 ## Release notes
 
-## 1.8
+### 1.9
 
-### 1.8.6
+#### 1.9.1
+
+**Improvements**
+
+- **[Helm Chart]** The operator will not create PersistentVolumeClaim (PVC) for log storage when you set `logStorageSize` to `0`.  [#398](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/398)
+- **[Operator]** The operator can detect the values of`mountPath` and `name` in `storageVolumes` are duplicated. An error is returned when duplicate values exist. [#388](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/388)
+- **[Operator]** The number of FE nodes can not be scaled down to 1. [#394](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/394)
+- **[Operator]** The values of the `feEnvVars`, `beEnvVars` and `cnEnvVars` fields in multiple values YAML files can be merged. [#396](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/396)
+- **[Operator]** Add `spec.containers.securityContext.capabilities` in the StarRocksCluster CRD to customize the Linux capabilities of containers. [#404](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/404)
+
+**Bug Fixes**
+
+Fixed the following issues:
+
+- **[Operator]** The `annotations` field in `service` can be updated. [#402](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/402) [#399](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/399)
+- **[Operator]** The modification of statefulset and deployment is patched instead of updated. It solves the problem that upgrading CN will cause all CN pods to be terminated and restarted when CN and HPA are enabled. [#397](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/397)
+- **[Operator]** The modification of service object is patched instead of updated. It avoids the operator to overwrite the modification, for example, when a Kubernetes Cloud provider is used and the service object is modified by that Kubernetes Cloud provider. [#387](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/387)
+- **[Operator]** Considered Cn Replicas when calculating component hash [#392](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/392)
+- **[Helm Chart]** The typo in `storageSpec` in [starrockscluster.yaml](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml) is correctified. [#385](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/385)
```

Comment:
correctified -> corrected
I don't think "correctify" is a popularly acceptable word. Use "correct" which can serve as a verb too.